src/main/java/com/northconcepts/datapipeline/examples/cookbook/ReadUuidFromPostgreSQL.java

```diff
@@ -0,0 +1,71 @@
+package com.northconcepts.datapipeline.examples.cookbook;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+
+import com.northconcepts.datapipeline.core.DataWriter;
+import com.northconcepts.datapipeline.core.StreamWriter;
+import com.northconcepts.datapipeline.internal.test.DataPipelineUnitTest;
+import com.northconcepts.datapipeline.jdbc.JdbcReader;
+import com.northconcepts.datapipeline.job.Job;
+
+public class ReadUuidFromPostgreSQL extends DataPipelineUnitTest {
+
+    final static String DATABASE_DRIVER = "org.postgresql.Driver";
+    final static String DATABASE_URL = "jdbc:postgresql://localhost:5432/contacts?&stringtype=unspecified";
+    final static String DATABASE_USERNAME = "test";
+    final static String DATABASE_PASSWORD = "test";
+    
+    public static void main(String[] args) throws Throwable {
+        Connection connection = null;
+        
+        try {
+            Class.forName(DATABASE_DRIVER);
+            connection = DriverManager.getConnection(DATABASE_URL, DATABASE_USERNAME, DATABASE_PASSWORD);
+            createTableAndInsertRecords(connection);
+            
+            JdbcReader reader = new JdbcReader(connection, "SELECT * FROM contacts");
+            DataWriter writer = StreamWriter.newSystemOutWriter();
+            
+            Job.run(reader, writer);
+        } finally {
+            if (connection != null) {
+                connection.close();
+            }
```

Comment:
We know this can't be null since `DriverManager.getConnection()` doesn't return null.

```suggestion
            connection.close();
```

Reply:
Done.